src/components/__tests__/skills.test.js

```diff
@@ -0,0 +1,63 @@
+import React from "react"
+import renderer from "react-test-renderer"
```

Comment:
Hello!
Could you rework this file to use `enzyme-to-json` library instead of `react-test-renderer`  ( there is no trace of it in master now)
You can look [here ](https://github.com/rahuldkjain/github-profile-readme-generator/blob/master/src/components/__tests__/work.test.js#L21-L27)- i assume it give you a hint

Reply:
Sure, fixed.